test/BaselineOutput/SingleRelease/SavePipe/TestParquetPrimitiveDataTypes-Data.txt

```diff
@@ -0,0 +1,21 @@
+#@ TextLoader{
+#@   header+
+#@   sep=tab
+#@   col=Id:I4:0
+#@   col=Timestamp:DZ:1
+#@   col=Message:TX:2
+#@   col=Data:TX:3
+#@   col=IsDeleted:BL:4
+#@   col=Amount:R4:5
+#@   col=TotalAmount:R8:6
+#@   col=Counter:I8:7
+#@   col=Amount2:R8:8
+#@   col=Flag:U1:9
+#@   col=Flag2:I1:10
+#@   col=Flag3:I2:11
+#@   col=Flag4:U2:12
+#@   col=Flag5:TS:13
+#@ }
+Id	Timestamp	Message	Data	IsDeleted	Amount	TotalAmount	Counter	Amount2	Flag	Flag2	Flag3	Flag4	Flag5
+1	"2000-01-01T01:01:01.0000000+00:00"	Record1	SomeData3	0	125.4	400	300000	3331313	3	-3	-600	600	"3100.00:00:00.1000000"
+1	"2000-12-31T23:59:59.9990000+00:00"	Record2	SomeData4	0	126.4	500	400000	4331313	4	-4	-700	700	
```

Comment:
>	 [](start = 96, length = 1)

The thing following this appears to be a missing value, from the fact that the field is blank. There must be something wrong *somewhere* it seems to me, since if it is not nullable, why is it mapping to the missing value?

Also there is the point that in your second datetime PR, if it *is* nullable, then this test will throw, and we ought to have some "happy path" test where we can just baseline the outputs of converting Parquet to test (e.g., `savedata data=parquet-file.parquet dout=output.txt`), which we cannot do if we are, by design, throwing exceptions. Both of these files at least at first glance seem to be converting to text files where there are missing values, either because there are missing values there, or else some fault in the parquet loader.

Reply:
Hi @TomFinley , It maps to missing when there is an overflow. Please see the below conversion code. Here the value isn't nullable. In DvDateTime code I have changed this function to set dst to default because we decided to not throw on reading missing values anywhere but rather set minimum or default and we also decided we could switch between default and minimum from type to type (i.e for int we use minimum and for boolean we use default) as long as we are consistent with throwing exception vs not throwing exception. 

            /// <summary>
            /// Converts a Parquet Interval data type value to a DvTimeSpan data type value.
            /// </summary>
            /// <param name="src">Parquet Interval value (int : months, int : days, int : milliseconds).</param>
            /// <param name="dst">DvTimeSpan object.</param>
            public void Conv(ref Interval src, ref DvTimeSpan dst)
            {
                try
                {
                    dst = new DvTimeSpan(TimeSpan.FromDays(src.Months * 30 + src.Days) + TimeSpan.FromMilliseconds(src.Millis));
                }
                catch (Exception ex)
                {
                    // Handle TimeSpan OverflowException
                    _ch.Error("Cannot convert Inteval to DvTimeSpan. Exception : '{0}'", ex.Message);
                    dst = DvTimeSpan.NA;
                }
            }

Reply:
Very well. As you said in an offline discussion this ought to be a configurable option, but possibly not as part of this PR, since we will have a chance to revisit post 0.5.